libs/ledger-live-common/package.json

```diff
@@ -218,6 +218,7 @@
     "lodash": "^4.17.21",
     "minimatch": "^5.1.0",
     "near-api-js": "^0.44.2",
+    "nock": "^13.0.5",
```

Comment:
Naive question, can't it be a `devDependencies`?

Reply:
Yes it can !

Reply:
Done